src/components/sbb-chip/sbb-chip.tsx

```diff
@@ -12,11 +11,11 @@ export class SbbChip extends LitElement {
 
   /** Size of the chip. */
   @property({ reflect: true })
-  public size: InterfaceSbbChipAttributes['size'] = 'xxs';
+  public size: 's' | 'xs' | 'xxs' = 'xxs';
```

Comment:
It doesn't matter, but I personally would invert the order

Reply:
Done